lib/plugin.js

```diff
@@ -118,9 +118,9 @@ export default fp(async function (
       await f.register(scriptsPn, { enabled: scripts, base: assetBase });
       await f.register(liveReloadPn, { development });
       await f.register(compressionPn, { enabled: compression });
+      await f.register(errorsPn);
       await f.register(assetsPn, { base, cwd });
       await f.register(dependenciesPn, { enabled: development, cwd });
-      await f.register(errorsPn);
```

Comment:
Error handler needs to be elevated in the register chain to deal with errors from the dependency plugin.